packages/terra-form-select/src/Select.scss

```diff
@@ -0,0 +1,118 @@
+:local {
+  .select {
+    background-color: #fff;
+    background-image: var(--terra-select-focus-animation-background, none);
+    background-position: 0 100%;
+    background-repeat: no-repeat;
+    background-size: var(--terra-select-focus-background-start, 0 0);
+    border-color: var(--terra-select-input-border-color, #dedfe0);
+    border-radius: 3px;
+    border-style: solid;
+    border-width: 1px;
+    cursor: pointer;
+    font-size: var(--terra-select-font-size, 1em);
+    line-height: var(--terra-select-line-height, 1.07);
+    padding: var(--terra-select-input-padding, 0.571em 0.5em);
+    transition-duration: var(--terra-select-focus-transition-duration, 0s);
+    transition-property: background-size;
+    transition-timing-function: ease;
```

Comment:
For form-textarea, we wanted to theme the timing-function. Should we do the same here? https://github.com/cerner/terra-core/blob/a6033d7e3670cea80497408167673168fd5d38f9/packages/terra-form-textarea/src/Textarea.scss#L25